jobs/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
ok, I didn't do that, I will add it to trello if it isn't there, so we don't forget.